Add buttons in header

pblayo opened the pull request:
Fixes #173 
Fixes #174 
Fixes #175

pblayo commented:
![image](https://user-images.githubusercontent.com/181349/49754759-3ac4f080-fcb7-11e8-94e9-6dc7f57bc42b.png)

fpagnoux commented on source/index.md:
```diff
@@ -1,4 +1,10 @@
-[![OpenFisca logo](https://openfisca.org/img/logo-openfisca.svg)](https://openfisca.org)
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" crossorigin="anonymous">
+<div class="button">
+  <a class="button_link" href="https://github.com/openfisca/openfisca-core"><i class="fab fa-github"></i> GitHub</a>
```
I think our conclusion during the planification was that a link towards https://github.com/openfisca would be best

pblayo replied:
Done.

sandcha commented on source/_static/style.css:
```diff
@@ -6,6 +6,41 @@
   padding-left: 10px;
 }
 
+.button {
```
> The generic name could be misleading as there are buttons in the documentation that are not using `.button` class. Set a more specific name or use this class as main buttons class and set only diff in children?

sandcha commented on source/index.md:
```diff
@@ -1,6 +1,19 @@
-
-> Download a [PDF offline version](https://media.readthedocs.org/pdf/openfisca/latest/openfisca.pdf) of this documentation.
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" crossorigin="anonymous">
+<div class="buttons-header">
```
🎉

sandcha commented:
Approving last changes. But, we need to fix this bug with "large" screens. 
Here is an example @pblayo (screen size above page):

![screen shot 2018-12-21 at 14 56 46](https://user-images.githubusercontent.com/6567910/50346087-84240400-0531-11e9-8653-31a69655c04c.png)

This screenshot comes from Firefox (menu: Tools > Web Development > Adaptative view).

sandcha commented:
> For `iPhone 6/7/8`, add some space between logo and buttons?
![screen shot 2018-12-21 at 14 57 27](https://user-images.githubusercontent.com/6567910/50346208-f399f380-0531-11e9-9d3b-57ddd4d89607.png)

pblayo commented:
@sandcha :
- for small screens, I've added spaces around the logo
- for large screens I've added some media queries, but there's no way to both have these buttons in the upper right corner and to manage every large screen properly.
I discussed the matter with Florian at the beginning and he strongly advocated to not choose to develop a template, which would be overkill and add maintainability cost.
